gui/mozregui/wizard.py

```diff
@@ -111,6 +111,27 @@ def __init__(self):
         self.ui.start_date.setDateTime(now.addYears(-1))
         self.ui.end_date.setDateTime(now)
 
+    def validatePage(self):
+        good_date = self.ui.start_date.date()
+        bad_date = self.ui.end_date.date()
+        current = QDateTime.currentDateTime().date()
+        if good_date < bad_date:
+            if bad_date <= current:
+                return True
+            else:
+                QMessageBox.critical(
+                    self,
+                    "Error",
+                    "Date you entered is larger than now,please try it again.")
+                return False
+        else:
+            QMessageBox.critical(
+                self,
+                "Error",
+                "Dates in good date field can't larger than bad date,\
+                please try it again.")
+            return False
```

Comment:
Could you put the "return False" at the end of the function, and remove the two others ? I think it will be more readable.